méli-mélo/th-empathy/empathy.html

```diff
@@ -92,3 +92,111 @@ <h4>Code</h4>
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Benefits and Programs</h2></a>
+					<p>Applying for benefits to which you may be entitled following a death.</p><br>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Representing someone who died</h2></a>
+					<p>The legal, financial, estate planning, and other aspects of representing someone who has died.</p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Taxes and finances related to death</h2></a>
+					<p>Information on managing finances of an individual who died, including preparing and filing their taxes.
+ </p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Estates and wills</h2></a>
+					<p>Find information about estate planning and preparing a will, as well as what to do when there is no will.</p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Death-related documents</h2></a>
+					<p>Documents you may need to obtain following a death.</p><br>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Contacting services</h2></a>
+					<p>Contact information for various federal or provincial programs, departments, or agencies.</p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<div class="col-xs-12 col-md-4 cards empathy quiz-content mrgn-bttm-lg">
+				<h2>Someone died, <br>what do I do?</h2>
+				<p class="mrgn-tp-lg">Answer a few simple questions for guidance on your situation.</p>
+				<p class="mrgn-tp-lg"><span class="fas fa-clock"></span> This will take less than a minute.</p>
+				<a href="#" class="mrgn-tp-lg btn btn-default btn-empathy btn-block btn-lg">Start questionnaire</a>
+
+				<img class="quiz-img mrgn-tp-lg center-block hidden-xs hidden-sm" src="./images/lloj-direction-cloud.256.png" alt="Someone died, what do I do?">
+		</div>
+	</div>
+</div>
+<h4>Code</h4>
+<pre><code>&lt;div class="mrgn-tp-lg container"&gt;
```

Comment:
Add the following div to the top of the page to style the code sample.
```suggestion
<div class="wb-prettify all-pre hide"></div>
<pre><code>&lt;div class="mrgn-tp-lg container"&gt;
```